### PR TITLE
feat(payment): PAYPAL-1625 added buy now for PayPalCommerceButtonStrategy

### PR DESCRIPTION
**Pull request by serhii-tkachenko**
## What?
...

## Why?
...

## Testing / Proof
...

### CODE REVIEW

File: packages/core/src/cart/cart-request-body.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * An object that contains the information required for creating cart.
+ */
+export default interface CartRequestBody {
```
Comment (animesh1987) [inline packages/core/src/cart/cart-request-body.ts:4]: Can we call this `BuyNowCartRequestBody`?

File: packages/core/src/cart/cart-request-body.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    locale?: string; // TODO: do I really need this one?
+    source?: string; // TODO: Update with data type
```
Comment (animesh1987) [inline packages/core/src/cart/cart-request-body.ts:6]:
```suggestion
    source: 'BUY_NOW';
```

File: packages/core/src/cart/cart-request-body.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/**
+ * An object that contains the information required for creating cart.
+ */
+export default interface CartRequestBody {
+    locale?: string; // TODO: do I really need this one?
```
Comment (animesh1987) [inline packages/core/src/cart/cart-request-body.ts:5]: maybe not :D

File: packages/core/src/cart/cart-request-body.ts (hunk 1 above)
Comment (serhii-tkachenko) [reply packages/core/src/cart/cart-request-body.ts:4]: Yes 👍

**Pull request closed without merging**